### ISSUE TITLE
add version to payload

### ISSUE THREAD

**Issue by tomkralidis** [opengeospatial/CoverageJSON#162]
It would be valuable for a CoverageJSON payload to identify conformance (similar to [OGC API - Records](https://github.com/opengeospatial/ogcapi-records/blob/master/core/examples/json/record.json#L3).  Example:

```
{
    "conformsTo": [
        "http://www.opengis.net/spec/covjson/1.0"
    ],
    "type": "CoverageType",
        "domain": {
            "type": "DomainType",
    ...
```

While [clause 10](https://opengeospatial.github.io/ogcna-auto-review/21-069.html#toc51) of the DP provides guidance on further qualifying the media type with conformance, would it also be valuable to add to the payload itself (to self identify independent of the transfer mechanism)?

**Comment by jonblower** [comment]
Yes, I think that's a sensible idea. This could maybe go into v1.1. I guess it would have to be optional or "strongly recommended" in order to preserve backward compatibility.

**Comment by tomkralidis** [comment]
+1.  PR in #163 (not sure if master branch is still 1.0 bound).

**Comment by chris-little** [comment]
@tomkralidis @jonblower We had better leave Master Branch as V1.0 until @ghobona confirms it is published and we can make master v1.1 (or V1.0.1).

**Comment by chris-little** [comment]
@ghobona @tomkralidis @jonblower I have created a V1.0 tag/release, so we can start developing CoverageJSON V1.1 on the Master branch. So, providing @ghobona has no objection, I propose we merge PR#163.

**Comment by havardf** [comment]
Since I am working on profiles for EDR, and our profile also adds requirements about response format, like CoverageJSON, I am curious how a profile plays into this. Will `conformsTo` allow for profile conformance classes? Or is this meant only for OGC specifications?

**Comment by tomkralidis** [comment]
My take is that if/once this is included in CoverageJSON, it would allow for additional conformance to downstream profiles.
